pages/faq/troubleshooting/troubleshooting.md

```diff
@@ -68,6 +70,14 @@ the firmware on the Raspberry Pi doesn't support channel 12 and 13.
 
 ### Error Notifications
 
+## Balena FIN
+
+### Balena FIN Internal eMMC/storage not showing up in Etcher (Linux systems) when trying to flash
+If you are trying to flash a Balena FIN device with a Balena image, and you are running a Linux based OS on your system you may experience the following error after you connect the Balena FIN to your system: The `Select target` button of the Etcher application does not list the Compute Module memory storage (although the `Select target` button is active and clickable).
+You will need to open a command-line terminal and navigate to where the balenaEtcher application is stored on your file system and start it from the command line:
+`# sudo ./balenaEtcher-1.5.95-x64.AppImage` (balenaEtcher version - 1.5.95 - may differ)
```

Comment:
This should probably at least be the current 1.5.116 ? @thundron @zvin

Reply:
How about no version, and we keep it generic. Such as, 
```suggestion
`$ sudo ./balenaEtcher-<VERSION>.AppImage`
```